servo-build-dependencies/ci.sls

```diff
@@ -0,0 +1,14 @@
+{% from 'common/map.jinja' import root %}
+{% from tpldir ~ '/map.jinja' import sccache %}
+
+sccache:
+  file.managed:
+    - name: {{ sccache.destination }}
+    - source: https://s3.amazonaws.com/rust-lang-ci/rust-ci-mirror/{{ sccache.version }}-sccache-{{ sccache.platform }}
```

Comment:
I wish, there's a real test failure on the cross build due to the Android SDK check I added, this looks like it might require some yak shaving. Investigating.